peret/__init__.py

```diff
@@ -281,9 +259,7 @@ def _get_id(entry: TagNode) -> str:
 
     """
     # pylint: disable=protected-access
-    return _strip_id(
-        entry._etree_obj.xpath('@xml:id')[0]
-    )
+    return _strip_id(entry._etree_obj.xpath("@xml:id").first)
```

Comment:
`.first` is not available here because i'm using `lxml` api directly...

Reply:
```suggestion
    return _strip_id(entry._etree_obj.xpath("@xml:id")[0])
```

Reply:
actually, how about this?
```suggestion
    return _strip_id(
        entry.attributes.get(f'{{{XML_NS}}}id')
    )
```

Reply:
as a matter of fact, Python 3.9 eventually brought string methods to remove prefixes and suffixes.
for older ones also `"tla1234"[len("tla"):]` does it as well.